### PR TITLE
Update GitHub Security guide with latest Starter Kit improvements

### PR DESCRIPTION
**Pull request by ingyhere**
## Purpose
- Minor text edits to promote clarity
- Update Dependabot with latest improvements from Starter Kits
## Proposed Changes
- [CHANGE] README contents
## Issues
- #110 
## Testing
- Locally tested with sample repository: full scan, pre-commit hook
- Text validated for consistency on Git web site

### CODE REVIEW

**Review by jpl-jengelke — APPROVED** [review]
Rather small PR with targeted, but impactful changes.

File: docs/guides/software-lifecycle/security/github-security/README.md [hunk 1]
```diff
@@ -48,14 +48,19 @@ If you do not have organizational permissions or if you wish to customize securi
...
      - We also recommend enabling Dependabot version updates _if you are using a package manager for your project_. This will help you keep your dependencies up-to-date. To configure Dependabot version updates:
        1. Create a `.github/dependabot.yml` file in your repository.
-       2. Specify the package-ecosystem, directory, and schedule for the updates. For example:
+       2. Specify the package-ecosystem, directory, schedule and branch to update. For example (from the [Slim Python Starter Kit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml)):
           ```yml
           version: 2
           updates:
-            - package-ecosystem: "npm"
-              directory: "/"
+            - package-ecosystem: 'pip'
```
Comment (jpl-jengelke) [inline docs/guides/software-lifecycle/security/github-security/README.md:55]: Explanation of changes: 
1. Tie back to Python Starter Kit
2. Add slightly more detail to configuration for a very routine request to control analysis time.
3. Limited comments to explain.

**Review by nutjob4life — APPROVED** [review]
Although I prefer the Oxford or "serial" comma, this looks legit 👍

File: docs/guides/software-lifecycle/security/github-security/README.md [hunk 2]
```diff
@@ -48,14 +48,19 @@ If you do not have organizational permissions or if you wish to customize securi
      - For added security, we suggest turning on Dependabot security updates to automatically generate pull requests for known vulnerabilities in your dependencies.
      - We also recommend enabling Dependabot version updates _if you are using a package manager for your project_. This will help you keep your dependencies up-to-date. To configure Dependabot version updates:
        1. Create a `.github/dependabot.yml` file in your repository.
-       2. Specify the package-ecosystem, directory, and schedule for the updates. For example:
+       2. Specify the package-ecosystem, directory, schedule and branch to update. For example (from the [Slim Python Starter Kit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml)):
```
Comment (riverma) [inline docs/guides/software-lifecycle/security/github-security/README.md:51]:
```suggestion
       2. Specify the package-ecosystem, directory, schedule and branch to update. For example, the below demonstrates a Python example from the [SLIM Python Starter Kit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml):
```

File: docs/guides/software-lifecycle/security/github-security/README.md (hunk 1 above)
Comment (riverma) [reply docs/guides/software-lifecycle/security/github-security/README.md:55]: Nice @ingyhere - much more useful to tie back to other SLIM projects and to do this for Python. Thanks!

**Review by riverma — CHANGES_REQUESTED** [review]

File: docs/guides/software-lifecycle/security/github-security/README.md (hunk 2 above)
Comment (jpl-jengelke) [reply docs/guides/software-lifecycle/security/github-security/README.md:51]: Excellent. Thanks.

**Pull request merged**